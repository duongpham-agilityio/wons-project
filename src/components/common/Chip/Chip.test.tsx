import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import ChipCustom from '.';

describe('Button component', () => {
  test('renders with default type', () => {
    const { container } = render(
      <ChipCustom color="success" size="sm">
        Chip Custom
      </ChipCustom>,
    );
    expect(container).toMatchSnapshot();
  });
});
