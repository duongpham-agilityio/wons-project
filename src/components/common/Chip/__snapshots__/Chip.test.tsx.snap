// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders with default type 1`] = `
<div>
  <div
    class="relative max-w-fit min-w-min inline-flex items-center justify-between box-border whitespace-nowrap h-6 text-tiny text-success-foreground bg-green-100 dark:bg-green-200 py-1.5 px-2 rounded-lg"
  >
    <span
      class="flex-1 px-1 text-ratio-900 dark:text-green-400 font-semibold text-base"
    >
      Chip Custom
    </span>
  </div>
</div>
`;
